File: utils/XyceCInterface/MatlabExamples/runACircuit/runACircuit.py
# This file is used to demonstrate the Python initialize(),
# runSimulation() and close() methods.

import sys

from xyce_interface import xyce_interface

# this calls the xyce_interface.open() method to
# make a xyce object

xyceObj = xyce_interface()

argv= ['runACircuit.cir']

result = xyceObj.initialize(argv)

result = xyceObj.runSimulation()

xyceObj.close()

File: utils/XyceCInterface/MatlabExamples/runACircuit/runACircuit.m




py.importlib.import_module('xyce_interface');

pyXyceObj = py.xyce_interface.xyce_interface()


argv=py.list({"runACircuit.cir"})

pyXyceObj.initialize(argv)

pyXyceObj.runSimulation()

pyXyceObj.close()